prime-router/docs/api/lookup-tables-openapi.yaml

```diff
@@ -0,0 +1,164 @@
+    email: usds@cdc.gov
+
+security:
+  - OktaAuth: [ ]
+
+paths:
+  /api/lookuptables:
+    get:
+      description: Fetch the list lookup tables
+      parameters:
+        - name: showAll
+          in: query
+          description: Set to true to include inactive tables
+          schema:
+            type: boolean
+      responses:
+        '200':
+          description: The list of tables with version information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TableVersionInfo'
+        '400':
+          description: if there was an error fetching the list of tables
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+
+  /api/lookuptables/{tableName}:
+    get:
+      description: Fetch the latest version of a table
+      responses:
+        '200':
+          description: The table data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DynamicTableRows'
+        '404':
+          description: The table name was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+        '400':
+          description: if there was an error fetching the table data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+    post:
+      description: Create a new version of a lookup table
+      requestBody:
+        description: Optional description in *Markdown*
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DynamicTableRows'
+      responses:
+        '200':
+          description: The table was created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TableVersionInfo'
+        '400':
+          description: if there was an error creating the table
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+
+  /api/lookuptables/{tableName}/{tableVersion}:
+    get:
+      description: Fetch a specific version of a table
+      responses:
+        '200':
+          description: The table data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DynamicTableRows'
+        '404':
+          description: The table name and/or version was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+        '400':
+          description: if there was an error fetching the table data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMsg'
+    post:
```

Comment:
I suggest that document that post requires PRIME admin rights, while get is open.